lib/ansible/modules/network/netconf/netconf_rpc.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2018, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+
+DOCUMENTATION = """
+---
+module: netconf_rpc
+version_added: "2.6"
+author:
+    - "Ganesh Nalawade (@ganeshrn)"
+    - "Sven Wisotzky (@wisotzky)"
+short_description: Execute operations on NETCONF enabled network devices.
+description:
+    - NETCONF is a network management protocol developed and standardized by
+      the IETF. It is documented in RFC 6241.
+    - This module allows the user to execute NETCONF RPC requests as defined
+      by IETF RFC standards as well as proprietary requests.
+options:
+  request:
```

Comment:
As per the discussed Netconf proposal https://github.com/ansible/proposals/issues/104 
The name of the operation is added as part of `rpc` option and the XML payload as `content` option. In this case, the entire XML is added as part of `request` option. 

IMO adding XML payload as part of `content` options makes it reusable across operations using Ansible variables and
operation as part of `rpc` option is more readable in the playbook